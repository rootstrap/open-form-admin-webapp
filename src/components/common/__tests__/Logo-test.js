import React from 'react';
import TestRenderer from 'react-test-renderer';

import Logo from '../Logo';

it('renders correctly', () => {
  const testRenderer = TestRenderer.create(<Logo />);
  expect(testRenderer.toJSON()).toMatchSnapshot();
});
