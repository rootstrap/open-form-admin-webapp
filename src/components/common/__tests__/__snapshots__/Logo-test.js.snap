// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  fill="none"
  height="21"
  viewBox="0 0 51 21"
  width="51"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M47.111 0L30.483 15.9948C27.8207 9.20742 20.5304 5.18777 13.0963 6.40841C5.66223 7.62905 0.170853 13.7474 0 21H5.50501C5.69649 15.8218 10.021 11.669 15.4045 11.4937C20.788 11.3183 25.3946 15.1802 25.9498 20.3341L25.8961 20.3964C25.9831 20.6279 25.9998 20.8219 26.0053 21H32.9745L51 3.70002L47.111 0Z"
    fill="#161631"
  />
</svg>
`;
